src/main/java/core/basesyntax/service/DataValidatorService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface DataValidatorService {
+    boolean isDataValid(List<String> data);
```

Comment:
Consider updating the method signature to include a specific return type, like 'List<String>', for better type safety and clarity.